code/app/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
+    id("com.android.application")
+}
+
+android {
+      namespace = "com.adobe.marketing.mobile.edge.consent.testapp"
+
+    defaultConfig {
+        applicationId = "com.adobe.marketing.mobile.edge.consent.testapp"
+        compileSdk = BuildConstants.Versions.COMPILE_SDK_VERSION
+        minSdk = BuildConstants.Versions.MIN_SDK_VERSION
+        targetSdk = BuildConstants.Versions.TARGET_SDK_VERSION
+        versionCode = BuildConstants.Versions.VERSION_CODE
+        versionName = BuildConstants.Versions.VERSION_NAME
+    }
+
+    buildTypes {
+        getByName(BuildConstants.BuildTypes.RELEASE)  {
+            isMinifyEnabled = false
+        }
+    }
+}
+
+dependencies {
+    implementation(project(":edgeconsent"))
+    implementation("com.adobe.marketing.mobile:core:2.+")
+    implementation("com.adobe.marketing.mobile:assurance:2.+")
+    implementation("com.adobe.marketing.mobile:edge:2.+")
+
+    implementation("androidx.appcompat:appcompat:1.5.1")
+    implementation("androidx.constraintlayout:constraintlayout:2.1.4")
```

Comment:
Are these dependencies automatically applied by the aepsdk-plugin?

Reply:
This is sample app, we try not to use aepsdk plugin, so in case the user copy our sample app code.  Same as why we keep the spotless here instead of using the plugin.